package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@viamrobotics/prime",
-  "version": "0.1.6",
+  "version": "0.1.7",
```

Comment:
I don't think we should bump the version since we're not actually affecting any code that is shipped to NPM (it's storybook stuff only)

Reply:
Cool. I was on the fence.